docs/build/references/command-line-interface.md

```diff
@@ -681,6 +684,22 @@ If true, database pruning of obsolete historical data will be enabled. Should be
 
 If true, enables preimages. Defaults to false.
 
+#### `offline-pruning-enabled` (boolean):
+
+If true, offline pruning will run on startup and block until it completes (approximately one hour on mainnet). This will reduce the size of the database by deleting old trie nodes.
```

Comment:
should we add: `The node will not be able to process blocks while offline pruning is ongoing`.?

Reply:
I think being more explicit is a good idea.